src/reanimated2/layoutReanimation/web/elementManager.ts

```diff
@@ -0,0 +1,297 @@
+import type { TransformsStyle } from 'react-native';
```

Comment:
What is the different between DOMManager and elementManager?

Reply:
`DOMManager` is responsible for inserting and removing animations in the document. `elementManager` performs operations on component that we want to animate. Initially there was only `DOMManager`, but to avoid import cycles I've decided to split it into two files.